examples/mongocxx/index.cpp

```diff
@@ -32,4 +32,28 @@ int main(int, char**) {
         db["restaurants"].create_index(index_spec, {});
         // @end: cpp-create-compound-index
     }
+
+    // Create a unique index.
+    {
+        // @begin: cpp-create-unique-index
+        document index_spec;
+        mongocxx::options::index index_options{};
+        index_spec << "cuisine" << 1;
```

Comment:
Maybe pick a different field for your unique index?  It's not a great example because many restaurants should be able to have the same type of cuisine.